src/lib/icons/BibleIcon.svelte

```diff
@@ -2,7 +2,7 @@
     export let _class = 'fill-black-100';
 </script>
 
-<svg class={_class} xmlns="http://www.w3.org/2000/svg" height="48" width="48"
```

Comment:
You need to resize this one and include the comments about it.  It is "Menu Book" from https://fonts.google.com/